build.gradle

```diff
@@ -307,7 +307,7 @@ dependencies {
 
     if (System.getenv("STIRLING_PDF_DESKTOP_UI") != "false") {
         implementation 'org.apache.commons:commons-compress:1.26.0'
```

Comment:
remove hard coded ``implementation 'org.apache.commons:commons-compress:1.26.0'`` version that was added due to security